tests/sentry/plugins/mail/activity/test_release.py

```diff
@@ -300,3 +300,67 @@ def test_no_committers(self):
         sent_email_addresses = {msg.to[0] for msg in mail.outbox}
 
         assert sent_email_addresses == {self.user3.email}
+
+    def test_uses_default(self):
```

Comment:
Do we have a test that checks that the org specific option overrides the default?

Reply:
added a user option so this is tested in `test_simple`